### ISSUE TITLE
Slider should render aria-valuetext property for reading the proper text

### ISSUE THREAD

**Issue by PorterNan** [OfficeDev/office-ui-fabric-react#3256]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? ( No)

Requested priority: (Normal) 

#### Describe the issue:

I am using slider and seems it cannot add the aria-valuetext for the slider, we don't exactly want to read out what we have in slider, we want read out the extra text for it

#### Actual behavior:

slider don't have a aria-valuetext property

#### Expected behavior:

slider should have it
